README.md

```diff
@@ -70,3 +70,73 @@ The newest version of freeplane (1.11.10) is needed.
 **There are no learning objectives or tests for an overview branch. The overview branch lets trainers pick and choose relevant topics within a branch to teach at any level required so it doesn’t need to complete the learning objectives that are strictly specified in the leaf sections.** 
 
 **In Overview, only the Header and Description are required.**
+
+## How to TAG using the skill tree
+
+Before showing examples, we are reiterating important rules and definitions of skills and the skill tree.
```

Comment:
Wouldn't it be better to have _one_ browsable document?

Reply:
Yes, you are right. We should put all of this information into the Wiki and link it here. We will do this after the merge, when we also update all the Markdown files for the wiki in one big leap.